dev-requirements.txt

```diff
@@ -1 +1 @@
-cloudify-common===4.5
+git+https://github.com/cloudify-cosmo/cloudify-common@master#egg=cloudify-common[dispatcher]==master
```

Comment:
If we are using 4.5 this is not necessary.

Reply:
Sure you are right